src/test/java/org/folio/rest/impl/UserTenantIT.java

```diff
@@ -59,13 +64,15 @@ public void beforeEach() {
   }
 
   @Test
+  @Order(9)
```

Comment:
please avoid this annotaiton in unit tests

Reply:
fixed